sysl2/sysl/parse/parse.go

```diff
@@ -127,8 +129,12 @@ func (p *Parser) Parse(filename string, fs afero.Fs) (*sysl.Module, error) {
 
 		for len(listener.imports) > 0 {
 			source = listener.imports[0]
+
 			listener.imports = listener.imports[1:]
 			if _, has := imported[source.filename]; !has {
+				if !p.rootFound && strings.HasPrefix(source.filename, "/") {
+					return nil, Exitf(AbsImportError, "error importing: import from root is not allowed if root is undefined")
```

Comment:
Didn't we talk about leaving this functionality to the filesystem? If root is found the chroot will forbid the import, if it isn't then the chroot will use the real / as root and allow. The parser should not be modified

Reply:
If this is implemented in chrootFs, that means chrootFs should be aware when it starts importing (so first module load should be ignored) and I thought that would be more complex since chrootFs is used a lot. Putting it in `Parse` seems to be simpler.